packages/js/src/core/user-info.ts

```diff
@@ -6,9 +6,16 @@ type Identity = {
   details?: Record<string, unknown>;
 };
 
-export type UserInfoResponse = IdTokenClaims & {
+type OrganizationData = {
+  id: string;
+  name: string;
+  description: string | undefined;
```

Comment:
Well I initially write it as `string | null` since that was also the type definition in Logto core, but the linter changed it to `string | undefined` automatically...